src/store/region-context.tsx

```diff
@@ -68,6 +70,8 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_eu = is_eu_location || isEuDomain()
         const is_non_eu = !is_eu
         const is_cpa_plan = cpa_plan_countries.includes(user_ip_country)
+        const first_two_countries = cpa_plan_countries.slice(0, 2)
+        const is_cpa_plan_no_bw = first_two_countries.includes(user_ip_country)
```

Comment:
You're already remove 'bw' from `cpa_plan_countries`. 
In this case `cpa_plan_countries` === `is_cpa_plan_no_bw`
Are we exactly need to rename and slice it?

Reply:
Yeah true, forgot to remove it thanks!